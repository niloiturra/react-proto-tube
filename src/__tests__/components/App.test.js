import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import App from '../../App';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('testing App component render', () => {
  it('should render App component correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
