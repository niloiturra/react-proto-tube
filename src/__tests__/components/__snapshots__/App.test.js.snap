// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing App component render should render App component correctly 1`] = `ShallowWrapper {}`;
